revolv/project/tests/tests.py

```diff
@@ -242,3 +244,92 @@ class ScrapeTest(TestCase):
     def test_scrape(self):
         result = scrape.delay()
         self.assertTrue(result.successful())
+
+
+class DonationAjaxTestCase(CreateTestProjectMixin, TestUserMixin, TestCase):
+    VALIDATE = 'payment/validate'
+    SUBMIT = 'payment/submit'
+
+    def setUp(self):
+        TestUserMixin.setUp(self)
```

Comment:
Should this have a call to `super`? tbh i'm not sure how super works with multiple inheritance